spring-xd-spark-streaming/src/main/java/org/springframework/xd/spark/streaming/DefaultSparkStreamingModuleOptionsMetadata.java

```diff
@@ -23,13 +23,13 @@
 import org.springframework.xd.module.options.spi.ModuleOption;
 
 /**
- * Default module options for {@link Processor}.
+ * Default module options for {@link org.springframework.xd.spark.streaming.java.Processor}.
```

Comment:
should probably link to the base type here (and mention that is the common parent for both java and scala Processor instances)